mmpose/structures/keypoint/transforms.py

```diff
@@ -62,3 +62,60 @@ def flip_keypoints(keypoints: np.ndarray,
         keypoints = [w, h] - keypoints - 1
 
     return keypoints, keypoints_visible
+
+
+def flip_regression(keypoints: np.ndarray,
+                    keypoints_visible: np.ndarray,
+                    flip_indices: List[int],
+                    center_mode: str = 'static',
+                    center_x: float = 0.5,
+                    center_index: int = 0):
+    """Flip human joints horizontally.
```

Comment:
I suggest replacing `flip_regression` with `flip_keypoints_custom_center`, which can be more readable and differentiated from `flip_keypoints()`